bin/webpack.js

```diff
@@ -1,19 +1,43 @@
 #!/usr/bin/env node
 
+const { exec } = require("child_process");
+const readline = require("readline");
+
+let io = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
 let webpackCliInstalled = false;
+
 try {
-	require.resolve("webpack-cli");
+	require.resolve("webpack-cli") &&
+	require("webpack-cli");
+} catch(err) {
 	webpackCliInstalled = true;
-} catch(e) {
-	webpackCliInstalled = false;
+	process.stdout.write("Sorry, 'webpack-cli' was not found inside this project.");
+
+	io.question("Do you want to install 'webpack-cli' automatically? [ Y/N ] \n", answer => {
+		if(/(y|yes)|(Y|Yes)/.test(answer)) {
+			process.stdout.write("Start downloading 'webpack-cli' package...\n");
+
+			exec("npm i webpack-cli -D ; npm link webpack", (err, data) => {
```

Comment:
This doesn't work cross platform. Windows doesn't support `;`.

Reply:
The user may use yarn instead. Detect `package-json.json` or `yarn.lock` and choose correct tool.

Please check if there is an existing package on npm that does this installing. I guess other tools do similar things.

Reply:
Also, what about `npx`? 😉

Reply:
Why do you link `webpack`?

Reply:
@ooflorent That‘s because i have an error message about unsuccessful webpack installation without ‘link‘ command running right after this installation,–
the main reason i went this way and not another.
 p.s. any suggestions for solving this problem would be highly appreciated.

Reply:
Error message: Cannot find module 'webpack/lib/Stats.js'